R/recipe.R

```diff
@@ -279,9 +279,11 @@ inline_check <- function(x) {
     cli::cli_abort(c(
       x = "No in-line functions should be used here.",
       i = "{cli::qty(length(funs))}The following function{?s} {?was/were} \\
```

Comment:
In order to not increase the number of `i` entries further, I opted to collapse the itemization onto the line where it's introduced. I believe this happens in a few other places in recipes already!